### PR TITLE
Add JSON schema describing the .pdfpc file format

### PR DESCRIPTION
**Pull request by fnevgeny**

### CODE REVIEW

File: schema/pdfpc.json [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      "type": "integer",
+      "minimum": 1,
+      "default": 20
+    },
+    "defaultTransition": {
+      "description": "Default slide transition",
+      "type": "string",
+      "$comment": "A proper regex pattern would be kilometer long...",
+      "default": "replace",
+      "examples": [
```
Comment (AndreasBilke) [inline schema/pdfpc.json:60]: I would leave examples within the man page. Otherwise you have to maintain these informations at two places.

Comment (fnevgeny) [reply schema/pdfpc.json:60]: But the man page is not versioned... The examples might change from one format version to another.

Comment (AndreasBilke) [reply schema/pdfpc.json:60]: No but the man page corresponds to the installed pdfpc version which is bound to a specific schema version

Comment (fnevgeny) [reply schema/pdfpc.json:60]: This again relates to whether the schema would be downloadable over the Internet. The local copy is always available in the tarball, of course. We can even `make install` it.

Comment (AndreasBilke) [reply schema/pdfpc.json:60]: I think I found my misunderstanding. Putting examples in the schema is a valid choice then.

**Comment by fnevgeny** [comment]
@AndreasBilke, it was awhile ago when we discussed the release mechanism <https://github.com/pdfpc/pdfpc/issues/605#issuecomment-920147831>. If it is fine with you, let's proceed.

**Comment by AndreasBilke** [comment]
@fnevgeny You mean the PR is ready for merge or that we continue working on that?

**Comment by fnevgeny** [comment]
> You mean the PR is ready for merge or that we continue working on that?

I think it's time to release v4.6, so no updates to the JSON schema are expected :)

**Comment by AndreasBilke** [comment]
Just to understand it correctly: This PR is ready for merge to release it together with v4.6?

**Comment by fnevgeny** [comment]
> Just to understand it correctly: This PR is ready for merge to release it together with v4.6?

Right. Unless you see a problem with it, of course.

**Review by AndreasBilke — APPROVED** [review]

**Pull request merged**